web/frontend/stylesheets/stylesheets.scss

```diff
@@ -205,3 +205,11 @@ td i.critical {
 .trento-pointer {
   cursor: pointer;
 }
+
+.align-right {
```

Comment:
Strange that such a tipical things are not already loaded and available...

Reply:
Don't tell me that :smile: I completely agree